homeassistant/components/waze_travel_time/config_flow.py

```diff
@@ -117,10 +136,16 @@ async def async_step_user(self, user_input=None) -> FlowResult:
                 {
                     vol.Required(
                         CONF_NAME, default=user_input.get(CONF_NAME, DEFAULT_NAME)
-                    ): cv.string,
-                    vol.Required(CONF_ORIGIN): cv.string,
-                    vol.Required(CONF_DESTINATION): cv.string,
-                    vol.Required(CONF_REGION): vol.In(REGIONS),
+                    ): TextSelector(),
+                    vol.Required(CONF_ORIGIN): TextSelector(),
+                    vol.Required(CONF_DESTINATION): TextSelector(),
+                    vol.Required(CONF_REGION): SelectSelector(
+                        SelectSelectorConfig(
+                            options=sorted(REGIONS),
+                            mode=SelectSelectorMode.DROPDOWN,
+                            translation_key=CONF_REGION,
+                        )
+                    ),
```

Comment:
One thing maybe: could this be moved to a constant?
The default could be handled with "add_suggested_values"

Reply:
Thank you @epenet